firmware/src/transceiver.c

```diff
@@ -906,16 +926,11 @@ void __ISR(AS_USART_ISR_VECTOR(TRANSCEIVER_UART), ipl6AUTO)
   if (SYS_INT_SourceStatusGet(g_hw_settings.usart_rx_source)) {
     if (g_transceiver.state == STATE_C_RX_IN_DUB ||
         g_transceiver.state == STATE_C_RX_DATA) {
-      if (UART_RXBytes()) {
-        // RX buffer is full.
-        PLIB_TMR_Stop(g_hw_settings.timer_module_id);
-        SYS_INT_SourceDisable(g_hw_settings.usart_rx_source);
-        SYS_INT_SourceDisable(g_hw_settings.usart_error_source);
-        PLIB_USART_ReceiverDisable(g_hw_settings.usart);
-        ResetToMark();
-        g_transceiver.result = T_RESULT_RX_INVALID;
-        g_transceiver.state = STATE_C_COMPLETE;
-      }
+      // It's impossible to overflow the buffer here, because each byte is 44uS
```

Comment:
Is there any reason to remove the code though, in case someone changes one of the variables? Or at least add a note where rdm_dub_response_limit is set.